packages/shared/src/styles/base.css

```diff
@@ -179,6 +179,12 @@ body {
   --theme-actions-plus-float: color-mix(in srgb, var(--theme-accent-bacon-bolder), transparent 92%);
   --theme-actions-plus-hover: color-mix(in srgb, var(--theme-accent-bacon-bolder), transparent 88%);
   --theme-actions-plus-active: color-mix(in srgb, var(--theme-accent-bacon-bolder), transparent 84%);
+  /* Help */
```

Comment:
You reverted my light mode fix :jenkins-triggered:
Please add this to light mode it's blocking

Reply:
Sorry looked a duplicate from change list, didn't see your commit